cpp/oneapi/dal/backend/primitives/selection/kselect_by_rows_simd.hpp

```diff
@@ -85,40 +83,36 @@ class kselect_by_rows_simd : public kselect_by_rows_base<Float> {
         const std::int64_t stride = data.get_shape()[1];
```

Comment:
line 79: what is the original type of `*result`? Why safe `detail::integral_cast` is not used?